Assets/MixedRealityToolkit/Attributes/MixedRealityServiceProfileAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Interfaces;
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Attributes
+{
+    /// <summary>
+    /// Attribute that defines which service a profile is meant to be consumed by.
+    /// Only applies to profiles that are consumed by types implementing IMixedRealityService.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true, Inherited = true)]
```

Comment:
AllowMultiple and Inherited should likely be false to avoid needing collision resolution.

```suggestion
    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
```